doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -148,38 +149,30 @@ Architecture they must be implemented as per the following specifications:
+|`ra2.ntw.005`|Multiplexer/meta-plugin|The networking solution deployed within the implementation may use a multiplexer/meta-plugin.|[`req.inf.ntw.06`](chapter02.md#23-kubernetes-architecture-requirements)<br><br>[`req.inf.ntw.07`](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.006`|Multiplexer/meta-plugin CNI Conformance|If used, the selected multiplexer/meta-plugin must integrate with the Kubernetes control plane via CNI.|[`req.inf.ntw.06`](chapter02.md#23-kubernetes-architecture-requirements)<br><br>[`req.inf.ntw.07`](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.007`|Multiplexer/meta-plugin CNI Plugins|If used, the selected multiplexer/meta-plugin must support the use of multiple CNI-conformant Network Plugins.|[`req.inf.ntw.06`](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.008`|SR-IOV Device Plugin for Network Intensive|When hosting workloads that match the Network Intensive profile and require SR-IOV acceleration, a Device Plugin for SR-IOV must be used to configure the SR-IOV devices and advertise them to the `kubelet`.|[`e.cap.013`](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
+|`ra2.ntw.009`|Multiple connection points with multiplexer/meta-plugin|When a multiplexer/meta-plugin is used, the additional non-default connection points must be managed by a CNI-conformant Network Plugin.|[`req.inf.ntw.08`](chapter02.md#23-kubernetes-architecture-requirements)<br><br>[`infra.net.cfg.004`](chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
+|`ra2.ntw.010`|User plane networking|When hosting workloads matching the Network Intensive profile, CNI network plugins that support the use of DPDK, VPP, or SR-IOV must be deployed as part of the networking solution.|[`infra.net.acc.cfg.001`](chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
+|`ra2.ntw.011`|NATless connectivity|When hosting workloads that require source and destination IP addresses to be preserved in the traffic headers, a CNI plugin that exposes the pod IP directly to the external networks (e.g. Calico, MACVLAN or IPVLAN) is required.|[`req.inf.ntw.14`](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.012`|Optional Device Plugins|When hosting workloads matching the Network Intensive profile that require the use of FPGA or other Acceleration Hardware, a Device Plugin for that FPGA or Acceleration Hardware may be used.|[`e.cap.016`](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
+|`ra2.ntw.013`|Dual stack CNI|The networking solution deployed within the implementation must use a CNI-conformant Network Plugin that is able to support dual-stack IPv4/IPv6 networking.|[`req.inf.ntw.04`](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.012`||||
```

Comment:
Delete the empty requirements.

Reply:
Done in https://github.com/cntt-n/CNTT/pull/2121/commits/9340d33ece604a3c39bb59dad4d63f18b6c2cb74